example/test/widget_test.dart

```diff
@@ -1,27 +1,30 @@
 // This is a basic Flutter widget test.
 //
 // To perform an interaction with a widget in your test, use the WidgetTester
-// utility that Flutter provides. For example, you can send tap and scroll
+// utility in the flutter_test package. For example, you can send tap and scroll
 // gestures. You can also use WidgetTester to find child widgets in the widget
 // tree, read text, and verify that the values of widget properties are correct.
 
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-import 'package:hypertrack_example/main.dart';
+import 'package:example/main.dart';
```

Comment:
What is the purpose of Example project? Do we need it at all?

Reply:
Doing rn, will update you soon in case of issues